solr/bin/solr

```diff
@@ -746,7 +750,6 @@ function run_tool() {
 
 # get status about any Solr nodes running on this host
 function get_status() {
-  CODE=4
```

Comment:
I dug around, and never saw where `CODE` was referenced.....  BATS still ran....

Reply:
A quick google for "bash and CODE" didn't turn up anything like it's a well known global variable or anything.  it also isn't used by other tools...

Reply:
My guess is it was supposed to be used in `exit` here - https://github.com/apache/solr/blob/11a86934f4c36f823c0a917fe7c5348b0aa96f57/solr/bin/solr#L966 but was never wired up correctly.

the idea being that exit codes for non successful commands would be set properly.

Reply:
a good reference on exit codes - https://www.cyberciti.biz/faq/linux-bash-exit-status-set-exit-statusin-bash/

Reply:
since we don't have this on other packageTools, can we leave adding it in as a seperate PR?    I don't have a ton of confidence in my bash skills, so nervous about tacking on a new feature and not getting it working...

Reply:
I've added https://issues.apache.org/jira/browse/SOLR-16794 to track properly doing Exit codes everywhere (and we'll want to bats test them!), so I'm going to call this PR good to go and merge it before it gets more stale?  Is that cool @risdenk ?